components/FeaturedData/FeaturedData.vue

```diff
@@ -111,23 +111,30 @@ h2 {
 }
 .data-wrap {
   align-items: center;
-  flex-wrap: wrap;
-  display: flex;
-  justify-content: center;
+  display: grid;
+  row-gap: 2rem;
```

Comment:
I think if the row gap is here, the margin size of the icons should be reduced, it gets pretty big otherwise and fills the whole page on mobile:
![previous](https://user-images.githubusercontent.com/37255664/166344555-e835ec71-1ca8-4cea-8eae-061dcd62c195.png)